modules/openapi-generator/src/main/resources/cpp-qt5-client/HttpRequest.cpp.mustache

```diff
@@ -285,7 +289,15 @@ void {{prefix}}HttpRequestWorker::execute({{prefix}}HttpRequestInput *input) {
     if (input->request_body.size() > 0) {
         qDebug() << "got a request body";
         request_content.clear();
-        request_content.append(input->request_body);
+        if(!isFormData && (input->var_layout != MULTIPART)){
```

Comment:
This whole block could be simplified I think (or did I miss something?)
(omitted brackets for shorter code, but I'm fine with them staying)
```
if(!isFormData && (input->var_layout != MULTIPART) && isRequestCompressionEnabled)
   request_content.append(compress(input->request_body, 7, {{prefix}}CompressionType::Gzip));
else
   request_content.append(input->request_body);
```